responsibleai/responsibleai/_tools/causal/causal_manager.py

```diff
@@ -0,0 +1,255 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
```

Comment:
why was it moved to _tools?

Reply:
@imatiach-msft I was trying out this new file structure. I personally like the idea of keeping abstract/base classes in one place and component-specific code in component-specific folders.

The benefit is that it's really clear we're breaking abstraction boundaries when component specific code goes into base classes. The most composable design will be one where the base classes reference each component only once in order to "register" them as valid managers.

I'm happy to change it back if you want to keep the current file structure.

Reply:
For the sake of getting the PR in faster I've moved it back.